addressfinder.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * The AddressFinder plugin for BigCommerce adds an autocomplete capability to
+ * the billing and shipping address fields of your online store.
+ *
+ * https://github.com/AbleTech/addressfinder-bigcommerce
+ */
 var fieldsForAddressType = {
     billing: { address_1: "FormField_8", address_2: "FormField_9", city: "FormField_10", country: "FormField_11", state: "FormField_12", postcode: "FormField_13" },
```

Comment:
Fairly confident.

1. Address fields are injected using what BigCommerce snippets that looks like `%%GLOBAL_ShipCustomFields%%` in the template files, and they are (afaik) uneditable
2. BigCommerce's own JS has a reference to `$('#FormField_11, #FormField_21').live(...blah...)`